chapters/packages.tex

```diff
@@ -345,9 +339,9 @@ \subsubsection{Example of Searching MODELICAPATH}\doublelabel{example-of-searchi
 directories. {[}\emph{\autoref{fig:roots} below shows an example} \lstinline!MODELICAPATH! =
 \lstinline!"C:\library;C:\lib1;C:\lib2"!\emph{,
 with three directories containing the roots of the package hierarchies}
-\lstinline!Modelica! \emph{,} \lstinline!MyLib!\emph{, and} \lstinline!ComplexNumbers!\emph{. The first two are represented as the subdirectories} \lstinline!C:\library\Modelica! \emph{and}
-\lstinline!C:\lib1\MyLib!\emph{, whereas the third is stored
-as the file} \lstinline!C:\lib2\ComplexNumbers.mo!\emph{.}
+\lstinline!Modelica! \emph{,} \lstinline!MyLib!\emph{, and} \lstinline!ComplexNumbers!\emph{. The first two are represented as the subdirectories} \filename{C:\\library\\Modelica} \emph{and}
```

Comment:
\\ inside \filename does not work as far as I can see.

Reply:
I mean double \

Reply:
Fixed.

On the other hand, shouldn't we follow the Modelica (MSL?) convention of using the forward slash as directory separator?